inc/namespace.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Altis\Security\Browser;
+
+use Altis;
```

Comment:
I think this ties the plugin to Altis. If we want it work outside of the software we can't depend on this.